client/app.js

```diff
@@ -0,0 +1,35 @@
+let form = document.getElementById("form");
+
+form.addEventListener("submit", handler)
+
+function handler (e) {
+    e.preventDefault();
+    const articleData = {
+        title: e.target.title.value,
+        user: e.target.author.value,
+        article: e.target.article.value,
+        //ts: Date.now(),
+    };
+
+    const options ={
+            method: 'POST',
+            body: JSON.stringify(articleData)
```

Comment:
not completely sure but does the options not need the header key and value as well?